src/pages/overview/OrganisationUnits.tsx

```diff
@@ -0,0 +1,58 @@
+import i18n from '@dhis2/d2-i18n'
+import React from 'react'
+import { OVERVIEW_SECTIONS, SECTIONS_MAP } from '../../lib'
+import { FilterAuthorizedSections, SummaryCard, SummaryCardGroup } from './card'
+import { OverviewGroup, OverviewGroupSummary } from './group'
+
+const TITLE = SECTIONS_MAP.organisationUnit.titlePlural
+
+export const OrganisationUnitsOverview = () => {
+    return (
+        <OverviewGroup title={TITLE}>
+            <OverviewGroupSummary>
+                {i18n.t(
+                    'Organisation units define the locations, or points of service, for which data can be collected. For example, they might be departments, offices, hospitals or clinics. Organisation units can be organised by group and group set.'
+                )}
+            </OverviewGroupSummary>
+            <OrganisationUnitsCardGroup />
+        </OverviewGroup>
+    )
+}
+
+export const OrganisationUnitsCardGroup = ({
+    showTitle,
+}: {
+    showTitle?: boolean
+}) => {
+    return (
+        <SummaryCardGroup
+            title={showTitle ? TITLE : undefined}
+            section={OVERVIEW_SECTIONS.organisationUnit}
+        >
+            <FilterAuthorizedSections>
```

Comment:
Do you think we should look at moving the filter logic `FilterAuthorizedSection` within `SummaryCardGroup` - so we don't have to add this everywhere?

Reply:
Sounds reasonable. I've moved `<FilterAuthorizedSections>` into `<SummaryCardGroup>` 👍